packages/shared/src/utils/days.spec.ts

```diff
@@ -33,6 +64,13 @@ describe('days', () => {
       expect(binaryDaysToDateArray(DaysOfTheWeek.Sunday | DaysOfTheWeek.Monday | DaysOfTheWeek.Tuesday | DaysOfTheWeek.Wednesday
       | DaysOfTheWeek.Thursday | DaysOfTheWeek.Friday | DaysOfTheWeek.Saturday)).to.eql([1, 2, 3, 4, 5, 6, 7]);
     });
+  });
+  describe('dayIsIn()', () => {
+    it('expect to return true', () => {
```

Comment:
I can't tell by reading the test description what is being tested.

Reply:
I know coming up with good test descriptions is not always easy.

Reply:
How about: "Saturday should be part of a weekend" and then modify slightly the test to test this scenario?